mock-tours.js

```diff
@@ -34,7 +34,7 @@ fs.mkdir(`${MOCK_PATH}/tour/details`, { recursive: true }, (err) => {
           // mock response of each hike/tour details
           fs.writeFile(
             `${MOCK_PATH}/tour/details/${tour.kuid}.json`,
-            JSON.stringify(tour, null, 2),
+            JSON.stringify({ tourDetails: tour }, null, 2),
```

Comment:
might want to wrap the call or the param in a try/catch because `JSON.stringify` can throw if the data is not JSONable... 
https://www.appmarq.com/public/security,1020102,Always-use-JSONparse-JSONstringify-with-trycatch-block-Javascript